Hymns/SceneDelegate.swift

```diff
@@ -19,12 +19,17 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         // Create the SwiftUI view and set the context as the value for the managedObjectContext environment keyPath.
         // Add `@Environment(\.managedObjectContext)` in the views that will need the context.
-        let rootView = HomeContainerView().environment(\.managedObjectContext, context)
+        let rootView = LaunchRouterView().environment(\.managedObjectContext, context).eraseToAnyView()
+        let rootViewTest = HomeContainerView().environment(\.managedObjectContext, context).eraseToAnyView()
+
+        var testing: Bool {
+            return ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil
+        }
```

Comment:
Why is this a computed variable? why not just

`var isTesting = ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil`

then just do
`let rootView = isTesting ? HomeContainerView().environment(\.managedObjectContext, context).eraseToAnyView() : LaunchRouterView().environment(\.managedObjectContext, context).eraseToAnyView()`

Reply:
It looks like this case doesn't cover UI tests

Reply:
Also, I would surround this whole thing with #IF DEBUG just in case